### DISCUSSION TITLE
Can 11ty publish from multiple folders?

### DISCUSSION THREAD

**Discussion by tjperry07** [11ty/eleventy#1743] [Q&A]
For example, I want `/docs` and `/v2.1/docs`

**Comment by pdehaan** [comment]
As far as I know, you can't publish to two output directories at once.
But you could build your site twice w/ two different commands and specify different `--output` directories https://www.11ty.dev/docs/config/#output-directory

So your package.json `scripts` might look something like this:

```js
"scripts": {
  "build:docs": "eleventy --output=docs",
  "build:vdocs": "eleventy --output=v2.1/docs",
  "build": "npm run build:docs && npm run build:vdocs"
}
```

Now when you run <kbd>npm run build</kbd> it should theoretically build the site once for `./docs/**` and another time for `./v2.1/docs/**`.

Probably not very efficient (depending on how much data fetching your site does), but maybe easier than setting up symlinks and stuff.

**Reply by Ryuno-Ki** [reply]
Either a symlink or some kind of reverse proxy on the server side.

**Reply by pdehaan** [reply]
Not sure how https://www.11ty.dev/docs/versions/ sets up their docs for each version...
You might be able to do that server side w/ redirects if you're using Netlify, like https://github.com/11ty/11ty-website/blob/master/netlify.toml

```toml
[[redirects]]
	from = "/docs/"
	to = "/v2.1/docs/"
	status = 301
	force = true
```

Next.js should have similar options: https://nextjs.org/docs/api-reference/next.config.js/redirects

Probably a few other ways to do it, but you'd have to remember to bump that `to`|`destination` value each time you release a new version.

**Comment by tjperry07** [comment]
I'll attempt this and report back. See, what I get.
